app/models/bulk_upload.rb

```diff
@@ -116,7 +116,6 @@ def prefix_namespace
 
   def unpend
     logs.find_each do |log|
-      log.skip_update_status = true
```

Comment:
We might still want to be able to skip in this case, otherwise we'll be recalculating the status and may as well not have cached it.

On the other hand, maybe it's good to be recalculating in case e.g. some validation has changed.

I'm not really sure here - any thoughts?

Reply:
I think recalculating makes it more accurate and we would avoid such bugs where something is marked completed, but actually isn't. But I think the issue would be speed, cause I don't think we put this on a queue, I'll update it and will add this back in

Reply:
Updated now!